.coderabbit.yml

```
@@ -0,0 +1,61 @@
+reviews:
+  path_instructions:
+    - path: "**/*.go"
+      instructions: |
+       
+         
+         "Verify that secret keys match the required format as below."
+         regex: "^[A-Za-z0-9+/]{32,64}$" 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)